goa/proj.android/libs/armeabi/gdb.setup

```diff
@@ -0,0 +1,2 @@
ocos2d/cocos/./../external/poly2tri D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/./../external/poly2tri/common D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/./../external/poly2tri/sweep D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/./../external/clipper D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/external/Box2D/.. D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/external/bullet/.. D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/ui/../editor-support D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/external/flatbuffers/.. D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/external/websockets/prebuilt/android/../../include/android D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/external/freetype2/prebuilt/android/../../include/android D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/external/freetype2/prebuilt/android/../../include/android/freetype2 D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/external/png/prebuilt/android/../../include/android D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/external/jpeg/prebuilt/android/../../include/android D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/external/tiff/prebuilt/android/../../include/android D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/external/webp/prebuilt/android/../../include/android D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/external/chipmunk/prebuilt/android/../../include D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/external/zlib/prebuilt/android/../../include D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/external/recast/.. D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/platform/android C:/Users/MAQUser/AppData/Local/Android/Sdk/ndk-bundle/build//../sources/android/cpufeatures D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/audio/android/../.. D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/audio/android/../../platform/android D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/./../external/edtaa3func D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/./../external/ConvertUTF D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/platform/android/.. D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/platform/android/../.. D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/storage/local-storage/../.. D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/network/../../external/curl/include/android D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/network/../../external/websockets/include/android D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/editor-support/cocosbuilder/../../2d D:/Git/MAQSoftware/GLEXP-Team-Chimple-goa-Hindi/goa/cocos2d/cocos/editor-support/cocostudio/WidgetReader
```

Comment:
@ankitk20: Should this be committed?

Reply:
It has local file path.

Reply:
This folder is not a part of standard Android .gitignore file